.teamcity/_self/bashNodeScript.kt

```diff
@@ -33,7 +33,7 @@ fun BuildSteps.bashNodeScript(init: ScriptBuildStep.() -> Unit): ScriptBuildStep
 	result.dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
 	result.dockerPull = true
 	result.dockerImage = result.dockerImage ?: "%docker_image%"
-	result.dockerRunParameters = result.dockerRunParameters ?: "-u %env.UID% --shm-size=1g"
+	result.dockerRunParameters = result.dockerRunParameters ?: "-u %env.UID%"
```

Comment:
This removes the default 1GB allocation of shared memory for every instance of `bashScriptNode`.

This is because the shared memory issue only arises in E2E testing and not in other steps that also use `bashNodeScript` (such as Unit Tests or environment setup steps).